meetings/10-06-2017/minutes.md

```diff
@@ -0,0 +1,81 @@
+# Collaborator summit at Node Interactive
+
+## Present
+* Peter: V8, worked on fixing regressions on micro benchmarks
+* Anil: Intel, works on standards for benchmarks (multiple languages)
+* Emily: Colorado, interested in test driven development
+* Michael Dawson: IBM 
+* Franzi (@fhinkel): V8, front-end benchmarks are easy to get (take popular websites), we 
+want to figure out, what are relevant benchmarks for Node
+* Hitesh: Microsoft, same problem as Franzi
+* Justin: On a different project, had runs every 15 minutes and posted them to PR
+* Anna (@addaleax): 
+* Suresh: Intel, works on micro-architecture performance, software optimization for runtimes
+
+Benchmarks should run in a reasonable time (not 24 hours).
+
+Benchmarks to determine if PRs that claim perf improvements really add anything
+
+Different scenarios of benchmarks: 
+VM, core, 
+
+[https://benchmarking.nodejs.org/](https://benchmarking.nodejs.org/) These graphs run nightly
+They helped catch the problem with hash-seed vulnerability that it wasn’t backported to the right branches :) 
+
```

Comment:
It was not that they were not backported to the right branches but that the backports were not correct.